lib/ansible/modules/utilities/logic/pause.py

```diff
@@ -40,6 +40,12 @@
       - Optional text to use for the prompt message.
     required: false
     default: null
+  echo:
+    description:
```

Comment:
version_added?

Reply:
I'll put 2.4 in the hopes we can get this into 2.4.x

Reply:
no, in that case , lets split PRs, get fix in 2.4, add echo in 2.5

Reply:
Will do. Ended up doing _a lot_ of polishing or how this behaves.